src/_includes/assets/css/screen.css

```
@@ -170,6 +182,135 @@ body {
 /* 5. Site Navigation
 /* ---------------------------------------------------------- */
 
+.exposed-button-nav {
+  color: var(--gray00);
+  background-color: var(--theme-color);
+  height: var(--header-element-size);
+  padding: 0 4px;
+  align-items: center;
+  min-width: var(--header-element-size);
+  border: 1px solid var(--gray00);
+  font-size: 18px;
+}
+
+@media (min-width: 601px) {
+  .exposed-button-nav {
+    padding-inline: 12px;
+  }
+}
+
+.exposed-button-nav:is(
+  :hover,
+  :hover:focus,
+  [aria-expanded='true'],
+  [aria-expanded='true']:hover
+),
+.lang-button-nav:is(
+  :hover,
+  :hover:focus,
+  [aria-expanded='true'],
+  [aria-expanded='false'].force-show
+) {
+  background-color: var(--gray00);
+  color: var(--theme-color);
+}
+
+.display-menu {
+  display: block !important;
+  max-height: calc(
+    100vh - var(--header-height) - var(--header-sub-element-size)
+  );
+  overflow-y: auto;
+  scrollbar-width: none;
+  text-align: left;
+  top: calc(var(--header-height) + var(--header-sub-element-size));
```

I think it would be best to have the dropdown cover the "Learn to code" banner entirely, rather than push the dropdown below the banner on mobile when the dropdown visible.

This could be simplified to the following:


```suggestion
  top: var(--header-height);
```

Also, we might not need the `header-sub-element-size` CSS variable at all if the dropdown covers up the banner.

---

And I think we should probably get rid of the donate button altogether and make it a link in the dropdown like on Learn.

Does that sound good to you, @ahmaxed?

Finally, it'd be nice to move the search bar to the mobile dropdown as well, but that may be outside the scope of this PR. If it is, that's something we can work on in another PR.

---

That  button gets loads of clicks. 
I should AB test it separately to see how it affects donations.